pkg/pillar/cmd/baseosmgr/handlecerts.go

```diff
@@ -318,7 +302,7 @@ func publishCertObjStatus(ctx *baseOsMgrContext, status *types.CertObjStatus) {
 	key := status.Key()
 	log.Debugf("publishCertObjStatus(%s)\n", key)
 	pub := ctx.pubCertObjStatus
-	pub.Publish(key, status)
+	pub.Publish(key, *status)
```

Comment:
Yes, it seems safer to pass a value and I added a check for that inside the Publish member function. Potentially that means we don't need to make a copy to save it. However, since go always passes embedded maps and slices by reference the code still creates a copy as part of saving it in the map in pubsub.